crates/oxc_minifier/src/printer/mod.rs

```diff
@@ -136,6 +141,17 @@ impl Printer {
         }
     }
 
+    fn print_space_before_identifier(&mut self) {
+        let ch = unsafe { str::from_utf8_unchecked(&self.code) }.chars().last();
```

Comment:
I am working on a new PR about [this line](https://github.com/Boshen/oxc/blob/b22c62de4f46a4230c7db9f5ea45cdd4d3c3d89f/crates/oxc_minifier/src/printer/mod.rs#L133)

I'll refactor this PR later.